pkgs/servers/x11/xorg/default.nix

```diff
@@ -702,13 +702,20 @@ lib.makeScope newScope (self: with self; {
     meta.platforms = stdenv.lib.platforms.unix;
   }) {};
 
-  libX11 = callPackage ({ stdenv, pkgconfig, fetchurl, xorgproto, libxcb, xtrans }: stdenv.mkDerivation {
+  libX11 = callPackage ({ stdenv, pkgconfig, fetchurl, fetchpatch, xorgproto, libxcb, xtrans }: stdenv.mkDerivation {
```

Comment:
I believe this goes in the `overrides.nix` file, as this is generated

Reply:
Yes, you're right! I was mildly concerned by the lack of fetchpatch in the file initially.